spec/models/miq_group_spec.rb

```diff
@@ -458,4 +467,35 @@
       expect(group.user_count).to eq(2)
     end
   end
+
+  describe "#single_group_users?" do
+    it "returns false if all users in the group belong to an additional group" do
```

Comment:
@jrafanie - all users need to belong to an additional group for the groups to be deleted. I can say 'every' user in the group instead of 'all'